pyproject.toml

```diff
@@ -9,7 +9,15 @@ description = "Differentiable wave optics library using JAX"
 readme = "README.md"
 requires-python = ">=3.10"
 license = {text = "MIT"}
-dependencies = ["jax >= 0.4.1", "einops >= 0.6.0", "flax >= 0.6.3", "chex>=0.1.5", "optax >=0.1.4", "scipy >= 1.10.0"]
+dependencies = [
+    "jax >= 0.4.1",
+ "einops >= 0.6.0",
+ "flax >= 0.6.3",
+ "chex>=0.1.5",
+ "optax >=0.1.4",
+ "scipy >= 1.10.0",
+ "ipykernel>=6.29.5",
```

Comment:
Not sure why @GJBoth's comment isn't showing up attached to line 19 here, but that ipykernel dependency is what we want to remove.